packages/client/src/containers/FavouritePage/FavouritePage.Container.js

```diff
@@ -12,8 +12,8 @@ const textObj = { sidebar: 'Simply Spices / Favourites', main: 'Favourites' };
 export const FavouritePage = () => {
   const [favourites, setFavourites] = useState([]);
   const [loading, setIsLoading] = useState(false);
-  const { isFavorite, updateFavoriteStatus } = useFavoriteService(true);
   const [sort, setSort] = useState('');
+  const { isFavorite, updateFavoriteStatus } = useFavoriteService();
```

Comment:
This isn't needed since each ProductCard component does this internally for its own favorite status